src/Pos/Client/Txp/Util.hs

```diff
@@ -120,18 +119,20 @@ prepareInpsOuts
     -> NonEmpty Address
     -> TxOutputs
     -> Either TxError (TxOwnedInputs Address, TxOutputs)
-prepareInpsOuts utxo addrs@(someAddr :| _) outputs = do
+prepareInpsOuts utxo addrs outputs = do
     futxo <- evalStateT (pickInputs []) (totalMoney, sortedUnspent)
-    let inputSum =
-            unsafeIntegerToCoin $ sumCoins $ map (txOutValue . toaOut . snd) futxo
-        newOuts
-            | inputSum > totalMoney =
-                TxOutAux (TxOut someAddr (inputSum `unsafeSubCoin` totalMoney)) [] <|
-                outputs
-            | otherwise = outputs
+    -- @pva701: @flyingleaf, we added fees and this code became invalid,
+    -- I just commented it. Remove these comments if it's ok or rewrite otherwise.
+    -- let inputSum =
+    --         unsafeIntegerToCoin $ sumCoins $ map (txOutValue . toaOut . snd) futxo
+        -- newOuts
+        --     | inputSum > totalMoney =
+        --         TxOutAux (TxOut someAddr (inputSum `unsafeSubCoin` totalMoney)) [] <|
+        --         outputs
+        --     | otherwise = outputs
```

Comment:
Ahem... elaborate on that, please. Maybe it's just me being stupid, but we have to have some change output no matter if we have fees or not. Where's that logic is handled now instead?